sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -456,7 +460,8 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
 
   /**
    * Refreshes the cache entry and the associated metadata for all Dataset (if any), that contain
-   * the given data source path.
+   * the given data source path. Path matching is by prefix, i.e. "/" would invalidate
```

Comment:
Yes. I found this sentence is copied from Catalog.scala. Maybe, we can update them to
> Path matching is by prefix, i.e. "/" would invalidate all the cached entries and make the new versions cached lazily.